src/components/taskDetails/Details.tsx

```diff
@@ -3,18 +3,26 @@ import setColor from './taskPriorityColors';
 import classNames from './task-details.module.scss';
 import { useRouter } from 'next/router';
 import { TaskDetailsProps } from '@/interfaces/taskDetails.type';
-import { GITHUB_LOGO } from '@/constants/url';
 
 const Details: FC<TaskDetailsProps> = ({ detailType, value }) => {
     const router = useRouter();
     const { query } = router;
     const isDevModeEnabled = query.dev === 'true' ? true : false;
     const color = value ? setColor?.[value] : undefined;
     const isGitHubLink = detailType === 'Link';
-    const openGitIssueLink = () => {
-        window.open(value, '_blank');
+
+    const gitHubIssueLink = isGitHubLink ? value : undefined;
+
+    const extractIssueNumber = () => {
+        const issueNumberMatch = value?.match(/\/issues\/(\d+)/);
```

Comment:
could you describe what this regex is matching `value?.match(/\/issues\/(\d+)/);`

Reply:
@gauravsinhaweb This regex matches the number after '/issues/'. For example, if html_url is 'https://github.com/sampleOrg/sampleRepo/issues/000', the regex will return '000'.

Reply:
could you define this regex in a constant with a descriptive name, and use it? so that it will be easier for devs working in the future on this to understand. It won't take much time.

Reply:
Sure. I have updated the code to use a constant. Thanks